src/utils.js

```diff
@@ -1,15 +1,41 @@
+import axios from "@nextcloud/axios";
+
 export const UserUtil = {
-	getName: document.querySelector("head").getAttribute("data-user"),
-	getDisplayName: document
-		.querySelector("head")
-		.getAttribute("data-user-displayname"),
+	getName: () => document.querySelector("head").getAttribute("data-user"),
```

Comment:
I think getUserName would be explicit and better fit here.

Reply:
Fixed in last commit